### PR TITLE
Some Windows fixes

### PR DESCRIPTION
**Pull request by nojb**

### CODE REVIEW

**Comment by let-def** [comment]
Thanks!

File: src/frontend/ocamlmerlin.c [hunk 1]
```diff
@@ -186,7 +187,11 @@ static void append_argument(unsigned char *buffer, size_t len, ssize_t *pos, con
   *pos = j;
 }
 
+#ifdef _WIN32
```
[the remark this answers is not in the document]
Comment (nojb) [reply src/frontend/ocamlmerlin.c]: Oops, indeed!

[the remark this answers is not in the document]
Comment (nojb) [reply src/frontend/ocamlmerlin.c]: Do you want to submit a PR yourself or shall I do it?

[the remark this answers is not in the document]
Comment (nojb) [reply src/frontend/ocamlmerlin.c]: https://github.com/ocaml/merlin/pull/857

**Pull request merged**